tests/test_cluster.py

```diff
@@ -1478,5 +1478,21 @@ def test_minimal(self):
                                                     }
                                                 })
 
+class TestClusterHbaseSnapshot(QdsCliTestCase):
+    
+    def test_snapshot(self):
+        sys.argv = ['qds.py', 'cluster', 'snapshot', '--label', '1234', '--parameters', ' {"s3_location":"myString", "backup_type":"full"}']
+        print_command()
+        Connection._api_call = Mock(return_value={})
+        qds.main()
+        Connection._api_call.assert_called_with('POST', 'clusters/1234/snapshot', {'parameters':  {'s3_location':'myString', 'backup_type':'full'}})
+
+    def test_restore_point(self):
+        sys.argv = ['qds.py', 'cluster', 'restore_point', '--label', '1234', '--parameters', ' {"s3_location":"myString", "backup_id":"abcd", "table_names":"tablename"}']
```

Comment:
Wondering whether it is a good idea to ask users to write complicated string on the command-line.